book/src/specification/textual-representation/concrete-syntax.md

```diff
@@ -68,7 +69,7 @@ Modules are lists of zero-or-more definitions. Definitions within a module must
 > <sub>Grammar:</sub>
 >
 > _item_ ::=\
-> &emsp;|&ensp;_alias-type-definition_\
+> &emsp;|&ensp;_constant-type-definition_\
```

Comment:
It feels like `constant-type-definition` is unbound -- should that be `constant-definition`?

Reply:
Yup, agreed